### PR TITLE
Various overall changes

### PR DESCRIPTION
**Pull request by Eric-Jackson**
Added some lists, more links, cleaned up some of the copy, added a header, etc.

### CODE REVIEW

**Review by Ben-MyBB — APPROVED** [review]
Looks good to me.

File: october-2017/17.md [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
 
 [![New MyBB responsive design](../resources/visit-thread-white.svg)](https://community.mybb.com/thread-213603-post-1286609.html#pid1286609)
 
-##Blog
-In regards to the MyBB Blog we are hoping to make more use of this by providing project updates and make more of the things we are working on behind the scenes public! We have created a repo on GitHub which everyone can view and open issues. If you have ideas for Blog posts that you would like to share with us open an issue on Github and we will take a look at those. Although this is a strange thing to use Github for we plan to use labels (accepted, rejected, under discussion) and we are definitely open to suggestions regardless if it is a series of blog posts or just a single post. You can find the [Github repo for this here](https://github.com/mybb/blog.mybb.com-drafts).
+### Blog
+Our blog will be getting some love as well. We're now creating our blog posts via a repository on GitHub, so you can help! (Or just get a sneak peak at what the next blog will be about) If you have an idea for a blog post that we haven't done yet, or think we could be doing something better, feel free to open an issue in the repository linked below.
```
Comment (euantorano) [inline october-2017/17.md]: The bracketed section feels like it should either be before the `!` or have a closing `.`. Not a grammar expert though 😉

File: october-2017/17.md [hunk 2]
```diff
@@ -1,20 +1,57 @@
...
+* Modern Design
+  <span style="font-size: 14px; margin-bottom: 10px; display: block; opacity: .7;">We are making sure to get the site up to modern design standards.</span>
+* Fully Responsive
+  <span style="font-size: 14px; margin-bottom: 10px; display: block; opacity: .7;">This is part of a larger goal to get all MyBB products responsive, we're starting with the main site.</span>
+* SEO
+  <span style="font-size: 14px; margin-bottom: 10px; display: block; opacity: .7;">Finding pages on MyBB should be a much better experience through search engines.</span>
+
+If you have some time and want to leave some feedback, or if you just want to look at the new design, please visit the following thread. Thanks!
 
 [![New MyBB responsive design](../resources/visit-thread-white.svg)](https://community.mybb.com/thread-213603-post-1286609.html#pid1286609)
```
Comment (euantorano) [inline october-2017/17.md]: Maybe also include a link to the `wip.mybb.com` repository?

File: october-2017/17.md [hunk 3]
```diff
@@ -1,20 +1,57 @@
...
+
+[![New MyBB responsive design](../resources/github-transparent.svg)](https://github.com/mybb/blog.mybb.com-drafts)
+
+### Official Demo Site
+You may or may not know this, but we keep the community forum as vanilla as possible so users can get a feel for what MyBB is, right out of the box. Unfortunately this limits some decisions we make where improving the community forum is concerned. Not only that, but it limits how users can explore a vanilla MyBB installation as well. We are going to be creating a demo website to solve both of these issues, allowing you as users to browse a clean installation of MyBB easily, (It will be missing a couple of features, for security reasons) as well as letting us make some quality of life improvements to our community forum. You can expect work to begin on this project after the new website has launched. Stay tuned for further updates!
```
Comment (euantorano) [inline october-2017/17.md]: `allowing you as users to browse a clean installation of MyBB easily,` - change the `,` to `.`.

> We are in the process of setting up a new demo system to solve both of these issues, allowing you as users to create and play with a clean installation of MyBB easily. Obviously these demo installs will be lacking a couple of features that a full blown install contains, such as the ability to install custom themes and plugins - this is purely a security precaution.
>
> Providing a clean demo system for users also opens up the possibility for more changes to be made to the official community forum - such as updating the styling to fit in with the rest of the website - in the future.
>
> You can expect work to begin on this project after the new website has launched. Stay tuned for further updates!

File: october-2017/17.md [hunk 4]
```diff
@@ -1,20 +1,57 @@
...
+
+### Extend Site Moderation Queue
+
+#### Approval Process
+It's no secret that getting a modification approved on the MyBB website can be a slow process. The main offender for this is our willingness to make sure submissions are safe and of good quality, which means exploring the code and installing them for ourselves. This just isn't a sustainable practice and results in a lot of frustration.
```
Comment (euantorano) [inline october-2017/17.md]: > The main offender for this is our mission to ensure that all submissions are safe and of good quality

File: october-2017/17.md [hunk 5]
```diff
@@ -1,20 +1,57 @@
...
+
+To speed up your addon submissions, we're going to be easing our policy for allowing submissions. Modifications will be placed into their own category when they're quickly submitted, listing them as "unapproved". We will still be able to manually go through and approve them, but it won't prevent you from having your mod listed on the site.
```
Comment (euantorano) [inline october-2017/17.md]: > Modifications will be placed into their own publicly viewable category when they're quickly submitted

(probably best to make it clear)

File: october-2017/17.md [hunk 6]
```diff
@@ -1,20 +1,57 @@
...
+
+If you find an issue with one of these unapproved addons, or even one that has been approved, please report it to us via the Private Inquiries forum so we can investigate properly.
+
+#### Paid Modification Support
+We are also taking a look into our support of paid modifications. In the past, these have been banned from being posted anywhere on the forum. Paid authors create some great things, so we'd like to embrace them more going forward. Our current solution is allowing paid modifications in the Requests/Services/Jobs forum, which you can find below. We will continue looking into ways to improve, so keep an eye out for that.
```
Comment (euantorano) [inline october-2017/17.md]: > We're also considering the possibility of adding thread prefixes to the Requests/Services/Jobs forum in an effort to make clear the kinds of services on offer. If you have any suggestions regarding prefixes you think could aid in the usability of this forum, we're all ears!

File: october-2017/17.md [hunk 7]
```diff
@@ -1,20 +1,57 @@
...
+
+![MyBB Poll results](./17-screen.png)
 
-##Official Demo Site
-It has long been one of our short falls that we haven't got a demo site set up. However, this is going to change. We currently have developed a back end script for this to be possible. Users can create a demo and this will be active for roughly 72 hours and will then expire. We have of course disabled certain features for security reasons and we are also providing a few well known plugins that you can install and see what MyBB can really do! After our new MyBB.com design has been completed our designers are going to work on the front end for our demo site so stay tuned for further updates on this!
+We have not decided on any specifics, but have discussed some of the following options:
```
Comment (euantorano) [inline october-2017/17.md]: > We have not decided upon any specifics regarding possible enhancements for the MyBB 1.x series, though some of the following possibilities have been discussed:
> 
> - Using the Twig templating engine to provide an improved templating experience, benefiting theme authors, plugin authors and the core.
> - Implementing support for the SASS styling system, allowing the use of variables and functions when crafting themes.
> - Providing a brand new response default theme with full mobile support for the modern web.
> - Many more user experience improvements, such as implementing common plugins into the core software.

**Review by euantorano — CHANGES_REQUESTED** [review]
Some minor changes and comments.

File: october-2017/17.md (hunk 2 above)
Comment (dvz) [reply october-2017/17.md]: If it's going to be up in a matter of days, we could leave it off

File: october-2017/17.md (hunk 1 above)
Comment (dvz) [reply october-2017/17.md]: The former, since we're using BE across the project

File: october-2017/17.md (hunk 2 above)
Comment (Eric-Jackson) [reply october-2017/17.md]: It certainly won't be up in days, though the base layout should be ready to start applying to other pages by then. I'll add a link in for that.

File: october-2017/17.md (hunk 7 above)
Comment (euantorano) [reply october-2017/17.md]: This comment still seems to stand @Eric-Jackson - not sure why it shows as outdated...

**Pull request merged**